src/aggregate.d

```diff
@@ -36,9 +36,9 @@ import ddmd.visitor;
 
 enum Sizeok : int
 {
-    SIZEOKnone, // size of aggregate is not computed yet
-    SIZEOKdone, // size of aggregate is set correctly
-    SIZEOKfwd, // error in computing size of aggregate
+    SIZEOKnone,             // size of aggregate is not computed yet
```

Comment:
I'm not a fan of these.  I would like to have dfmt automatically do the alignment, but I think manually maintaining them is a waste of time.